docs/library/cli/et.md

```diff
@@ -0,0 +1,29 @@
+## `et` 👽
+
+`et` is a CLI tool that helps with routine tasks. Invoke using `uv run et`.
+
+## ⬛ `et make`
+
+Groups several commands that generates project files for you. It includes:
+
+### 1. `et make test`
+
+Creates a new specification test file for an EIP.
+
+Usage: `et make test [OPTIONS]`
```

Comment:
Could you take a stab at using the pattern applied in the `evm_bytes` doc to grab the docstrings from `et` and sub-commands and include them in the doc?

The output is not perfect, but it's the most sustainable way to maintain our docs, imo.

https://github.com/ethereum/execution-spec-tests/blob/5cf1f24b39ddef259470f58413337b81bda081a6/docs/library/cli/evm_bytes.md?plain=1#L1-L6

Reply:
Here's the result:
https://ethereum.github.io/execution-spec-tests/main/library/cli/evm_bytes/

As I said, the result's not perfect, so if you prefer to keep it simple, that's also fine. But mid-term I'd like to find a solution that works well for this.

Reply:
Done!